package.json

```diff
@@ -25,15 +25,19 @@
     "babel-loader": "^7.1.4",
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-2": "^6.24.1",
     "coveralls": "^3.0.0",
     "enzyme": "^3.3.0",
+    "enzyme-adapter-react-16": "^1.1.1",
     "enzyme-react-16-adapter-setup": "^0.1.0",
     "eslint": "^4.18.2",
     "eslint-config-airbnb": "^16.1.0",
     "eslint-plugin-import": "^2.9.0",
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-react": "^7.7.0",
+    "fetch-mock": "^6.3.0",
```

Comment:
these mocks are interesting but Jest does all this natively. Any reason we should expand to use these?

Reply:
I went down the route of using a tutorial and went with this API.  If you or Aric are free Friday, or sometime next week, I'd be happy to live pair and go over Jest testing in general.

In the interest of getting some of the client codebase out for the team to have something visible to work off of, how about I create a TODO/Trello card to refactor this for next week?

Reply:
Sounds solid. I'd throw a card for tech debt and we can get to it at some point. Just something to think about since our modules are growing rather rapidly for a small project.